data/txHandler.go

```diff
@@ -203,6 +204,8 @@ func MakeTxHandler(opts TxHandlerOpts) (*TxHandler, error) {
 				uint64(opts.Config.TxBacklogAppTxPerSecondRate),
 				time.Duration(opts.Config.TxBacklogServiceRateWindowSeconds)*time.Second,
 			)
+			// set appLimiter triggering threshold at 50% of the base backlog size
+			handler.appLimiterBacklogThreshold = int(float64(opts.Config.TxBacklogSize) * float64(opts.Config.TxBacklogRateLimitingCongestionPct) / 100)
```

Comment:
Let me just make sure I have this right: the reason this condition is different from before is because `congestedERL` measures traffic against `cap(handler.backlogQueue)`, which will be larger than `opts.Config.TxBacklogSize` if ERL is enabled. So `congestedARL` uses `opts.Config.TxBacklogSize` and ignores the increase to the backlog size due to ERL?

Reply:
this is correct. From the metrics we observed during high app load the backlog size was hovering around 20k (with cap of 74k) that did not allow ARL to kick in. The new threshold (13k vs 37k) should kick in ARL earlier decrease load from too frequent apps.